lib/Model/DeckBoard.php

```diff
@@ -106,7 +106,7 @@ public function getOwner(): string {
 	/**
 	 * @param int $lastModified
 	 *
-	 * @return DeckShare
+	 * @return DeckBoard
```

Comment:
```suggestion
```
Already in the function declaration as well

Reply:
we already had a discussion about this, we keep phpdoc